plugins/pay.c

```diff
@@ -1708,6 +1739,11 @@ static void add_new_entry(struct json_stream *ret,
 		json_add_tok(ret, "label", pm->label, buf);
 	if (pm->preimage)
 		json_add_tok(ret, "preimage", pm->preimage, buf);
+
+	if (pm->amount != NULL)
+		json_add_string(ret, "amount_msat",
+				fmt_amount_msat(tmpctx, &pm->amount_sent));
```

Comment:
Hmm, this will print 0 for failed payments (since it's allocated and never updated).

Probably need to gate this on "status" != "failed"?